src/sass/main.scss

```diff
@@ -75,3 +77,9 @@ body > .au-body {
 .au-footer .au-link-list li {
 	@include AU-space( padding-right, 1unit );
 }
+
+.au-side-nav {
```

Comment:
Nice work @Xochi, for this one I would just change the grid in the HTML. We can reduce the column size and use the offset class.

https://designsystem.gov.au/components/grid-12/#offset